jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -826,11 +833,21 @@ pipeline {
             node(AGENT_X64) {
                 checkout scm
                 script {
+                    def findFailedPlugins = buildMessage(search: 'Failed plugins')
```

Comment:
Is there a better way than search for this text?
e.g. a flag set by the lib or similar?
Or just return code of the `build.sh` run in the python code through jenkins lib?

Reply:
Not a better way! Python code returns 0 as exit code and we want to flag is as unstable.

Reply:
I am not very convinced by this method.
But giving the circumstances I can approve it for now.
We need to improve this to some degree later.

Thanks.